homeassistant/components/sonos/services.yaml

```diff
@@ -63,6 +63,11 @@ remove_from_queue:
           max: 10000
           mode: box
 
+get_queue:
+  target:
+    entity:
+      domain: media_player
```

Comment:
Shouldn't we filter only Sonos integration entities too?

Reply:
Yes, great catch.  Update in this PR #124815